data.sql

```diff
@@ -8,4 +8,22 @@ INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts)
 VALUES ('Pikachu', '2021-01-07', 15.04, FALSE, 1);
 
 INSERT INTO animals (name, date_of_birth, weight_kg, neutered, escape_attempts) 
-VALUES ('Devimon', '2017-05-12', 11, true, 5);
\ No newline at end of file
+VALUES ('Devimon', '2017-05-12', 11, TRUE, 5);
```

Comment:
Please add the queries in the `queries.sql` file for following mentioned scenarios:

Please UPDATE the animals table by setting the species column to unspecified, then roll back the change. For
example:
  `BEGIN; -- start transaction
   UPDATE animals
   SET species = 'unspecified'; -- make change
   SELECT species from animals; -- verify that change was made
   ROLLBACK;
   SELECT species from animals; -- verify that change was undone`

Please UPDATE the animals table by setting the species column to digimon for all animals that have a name ending in mon.
please UPDATE the animals table by setting the species column to pokemon for all animals that don't have species already set. For example:
`  BEGIN;
   UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon'; 
   UPDATE animals SET species = 'pokemon' WHERE species IS NULL; 
   SELECT species from animals; -- verify that change was made
   COMMIT;
   SELECT species from animals; -- verify that change persists after commit
`
Please delete all records in the animals table, then roll back the transaction. For example:
 ` BEGIN;
   DELETE FROM animals WHERE ;
   SELECT COUNT(*) FROM ANIMALS;
   ROLLBACK;
   SELECT COUNT(*) FROM ANIMALS;
`
Please DELETE all animals born after Jan 1st, 2022
please Create a savepoint for the transaction
Please UPDATE all animals' weight to be their weight multiplied by -1.
Then Rollback to the savepoint
Please UPDATE all animals' weights that are negative to be their weight multiplied by -1 and then Commit the transaction. For example 
  ` BEGIN;
    DELETE FROM animals WHERE  date_of_birth >= '2022/01/01' ;
    SAVEPOINT SP1;
    UPDATE animals SET weight_kg = weight_kg * -1; 
    ROLLBACK TO SP21;
    UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0; 
    COMMIT;
`
Please create queries to answer following questions:
` How many animals are there?
How many animals have never tried to escape?
What is the average weight of animals?
Who escapes the most, neutered or not neutered animals?
What is the minimum and maximum weight of each type of animal?
What is the average number of escape attempts per animal type of those born between 1990 and 2000?
`

Reply:
Hi @SajjadAhmad14 
THank you for your feedback.
I've done all of theses requirements, but I encountered a strange issue that I didn't find all my commit in the repo. I didn't know the reason but I pushed again after make some informal changes.
Sorry again for this copy.